### DISCUSSION TITLE
Reverted Pull Requests

### DISCUSSION THREAD

**Discussion by adjurdjevic** [rfennell/AzurePipelines#1945] [Q&A]
Is it possible to remove work item from a Release Note list if Pull Request is reverted?

**Comment by rfennell** [comment]
Currently the answer is no. 

The task does an API call to get the [PR details](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-requests/get-pull-request?view=azure-devops-rest-7.1#gitpullrequest), the contents of which is exposed to the task's handlebar template. The problem is that is response does not appear to contain any details of whether the PR was reverted.

So to get the information to enable you to remove any reverted PR from the release notes would require extra API calls (for each PR) that I currently assume will need to check the [revert status API endpoint](https://learn.microsoft.com/en-us/rest/api/azure/devops/git/reverts/get-revert?view=azure-devops-rest-7.1#gitrevert).
